packages/webapp/src/elections/components/ongoing-election-components/ongoing-round/round-info/meeting-link/meeting-link.tsx

```diff
@@ -18,7 +18,7 @@ import { useEncryptionPassword, usePasswordModal } from "encryption";
 
 import MeetingButtons from "./meeting-buttons";
 import MeetingLinkModal from "./meeting-link-modal";
-import { election as electionEnvVars } from "config";
+import { election as electionConstants } from "config";
```

Comment:
nit: not big deal, but maybe `electionConfig`? `constants` appear that these values are set in stone, whereas we actually allow the community to set their own election params...

Reply:
For now, I'll switch back to `electionEnvVars`. It's not great, but `electionConfig` is a variable already declared in this file. This will likely change as the refactor continues.